tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/ManualInstrumentation/FeatureFlags/FeatureFlagsSdkEvaluateIntegration.cs

```diff
@@ -0,0 +1,50 @@
+// <copyright file="FeatureFlagsSdkEvaluateIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Threading;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.DuckTyping;
+using Datadog.Trace.FeatureFlags;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Datadog_Trace_Manual;
+
+/// <summary>
+/// Datadog.Trace.FeatureFlags.IEvaluation Datadog.Trace.FeatureFlags.FeatureFlagsSdk::Evaluate(System.String,System.Type,System.Object,Datadog.Trace.FeatureFlags.IEvaluationContext) calltarget instrumentation
+/// </summary>
+[InstrumentMethod(
+    AssemblyName = "Datadog.Trace.Manual",
+    TypeName = "Datadog.Trace.FeatureFlags.FeatureFlagsSdk",
+    MethodName = "Evaluate",
+    ReturnTypeName = "Datadog.Trace.FeatureFlags.IEvaluation",
+    ParameterTypeNames = [ClrNames.String, "Datadog.Trace.FeatureFlags.ValueType", ClrNames.Object, ClrNames.String, "System.Collections.Generic.IDictionary`2[System.String,System.Object]"],
+    MinimumVersion = "3.31.0",
+    MaximumVersion = "3.*.*",
+    IntegrationName = nameof(IntegrationId.DatadogTraceManual))]
+[Browsable(false)]
+[EditorBrowsable(EditorBrowsableState.Never)]
+public sealed class FeatureFlagsSdkEvaluateIntegration
+{
+    internal static CallTargetState OnMethodBegin<TTarget, TTargetType>(ref string flagKey, ref TTargetType targetType, ref object? defaultValue, ref string? targetingKey, ref IDictionary<string, object?>? attributes)
+    {
+        return new CallTargetState(null, new State(flagKey, (FeatureFlags.ValueType)(Convert.ToInt32(targetType)), defaultValue, targetingKey, attributes));
```

Comment:
I'm still not sure why you need the `Convert.ToInt32`, I'm pretty sure the calltarget code supports just using `int` directly, but 🤷‍♂️ it's just a minor perf hit, and we can always handle it later if we want to

Reply:
I tried and reverted the change because I set the parameter as int (and the runtime did not like it)... :_